bitbots_docs/docs/manual/tutorials/cl_simulation_testing_setup.rst

```diff
@@ -0,0 +1,70 @@
+User CL Simulation Testing Setup
+================================
+
+This setup is very similar to the tutorial on setting up a local environment for
+development documented in our `documentation <https://docs.bit-bots.de/meta/manual/tutorials/install_software_ros2.html>`_.
+
+As such you can lookup some of the needed requirements there.
+
+**0. Requirements**
+
+- have an LDAP mafiasi account for access to the CLs
+- have ros2 aliases setup (see linked docs)
+- have github ssh access setup for bitbots_meta (see linked docs)
+
+**1. Setup and download our software**
+
+- SSH into the ``CL0*`` with your mafiasi user
+- setup bitbots_meta on nvme nas share
+
+.. code-block:: bash
+
+  cd /srv/ssd_nvm/
+  mkdir -p "$USER/colcon_ws/src"
+  cd "$user/colcon_ws/src"
+  git clone git@github.com:bit-bots/bitbots_meta.git && cd bitbots_meta
+  make pull-init
+
+- setup a python virtualenv on the nvme share for faster ``pip`` package installation
+
+.. code-block:: bash
+
+  cd "/srv/ssd_nvm/$USER/colcon_ws"
+  python -m virtualenv env
```

Comment:
The env inside of colcon_ws doesn't seem to be a good idea; colcon tries to build some of the stuff inside of the env.

Reply:
Change to now setup the venv in the ssd_nvm `$USER` dir